<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalAlign</class>
 <widget class="QFrame" name="GlobalAlign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="resolution_spin_box">
       <property name="prefix">
        <string>Resolution:</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="load_down_sampled">
       <property name="text">
        <string>LoadDownSampled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="align_each_other">
       <property name="text">
        <string>AlginEachOther</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="align_up">
       <property name="text">
        <string>AlignUp</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
